types/ldapjs/index.d.ts

```diff
@@ -309,6 +309,8 @@ export interface Server extends EventEmitter {
      */
     listenFD(fileDescriptor: any): void;
 
+    close(callback: () => void): this;
```

Comment:
My mistake. This method is wrapped, and cannot accept a callback at all. If you remove the callback, I can approve this for merge.

Reply:
But what about https://github.com/ldapjs/node-ldapjs/blob/master/lib/server.js#L693 ? It looks like it's full passing-through the method to `(net|tls).Server`. Furthermore, the callback actually gets called in the project where I use it.

Reply:
I am so sorry. You are correct. I was inadvertently looking at a different version of ldapjs where this callback function is not present.

Reply:
np thank you for everything! :)